backend/package.json

```diff
@@ -7,16 +7,20 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "ts-node-dev index.ts",
     "tsc": "tsc",
-    "postinstall": "prisma generate"
+    "postinstall": "prisma generate",
+    "prismaMigrate": "npx prisma migrate dev --name"
```

Comment:
does this command work as is without having to provide a path to the .env file? i know some other members had issues with this

Reply:
Yeah, it does work for me without providing a path to .env. I don't quite understand by what you mean by having to provide a path to the .env file, can you elaborate on this?